### PR TITLE
get profiles for primary ac and show them in the secondary ac tab

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

**Comment by melisabok** [comment]
Please don't merge this. I'm planning to show the secondary AC name in the Assigned Papers tab.

**Comment by zbialecki** [comment]
It looks pretty good to me. I don't see anything specific I would change. Is this the PR you needed to modify the templates for?

**Comment by melisabok** [comment]
Yes, I need to add the secondary AC name in the Handlebars.templates.noteAreaChairs

**Comment by melisabok** [comment]
@zbialecki can you review this again? it should be ready now.

File: openreview/conference/templates/areachairWebfield.js [hunk 1]
```diff
@@ -762,12 +812,16 @@ var renderTableRows = function(rows, container, secondary_meta) {
...
-    Handlebars.templates.noteMetaReviewStatus
+    secondary_meta ? Handlebars.templates.noteAreaChairs : Handlebars.templates.noteMetaReviewStatus
   ];
 
   var rowsHtml = rows.map(function(row) {
     return row.map(function(cell, i) {
-      return templateFuncs[i](cell);
+      var html = templateFuncs[i](cell);
+      if (!secondary_meta && i == 4 && row[4].secondaryAC) {
+        html = html + '<br><p><strong>Secondary Area Chair: </strong><br>' + row[4].secondaryAC.name + '<span class="text-muted">(' + row[4].secondaryAC.email + ')</span></p>';
```
[the remark this answers is not in the document]
Comment (melisabok) [reply openreview/conference/templates/areachairWebfield.js:822]: Ohhh I thought that was the way to do it 🤷‍♀️

[the remark this answers is not in the document]
Comment (zbialecki) [reply openreview/conference/templates/areachairWebfield.js:822]: I didn't mean hack in a bad way!

**Comment by melisabok** [comment]
Show the secondary meta review now:

Primary tab:
Screen Shot 2020-06-19 at 9 22 54 AM

Secondary tab:
Screen Shot 2020-06-19 at 9 23 02 AM

**Comment by melisabok** [comment]
I'm not sure if I want to merge this PR into master, it is too hacky and I can save it for reference if we need it for ICLR.

**Comment by melisabok** [comment]
I will close this PR for now. We may use it for ICLR. We can reopen it later.

**Pull request closed without merging**